docs/CONTRIBUTING.md

```diff
@@ -84,7 +80,8 @@ void CollisionCheck_SpawnWaterDroplets(GlobalContext* globalCtx, Vec3f* v);
 ```
 
 Before PRing with a `NON_MATCHING`, you can try
-- using the [decomp-permuter](tools/decomp-permuter) to find a closer match,
+
+- using the [decomp-permuter](../tools/decomp-permuter) to find a closer match,
```

Comment:
We no longer have `decomp-permuter` in tools. Probably better to link to the github repo